src/util/system.cpp

```diff
@@ -359,6 +374,19 @@ Optional<unsigned int> ArgsManager::GetArgFlags(const std::string& name) const
     return nullopt;
 }
 
+bool ArgsManager::GetCommand(std::string& cmd, std::vector<std::string>& args) const
```

Comment:
Might be good to call it `AddCommand` and `GetCommand` for consistency?

Reply:
I took the naming from https://github.com/ajtowns/bitcoin/commit/ff48eb93bd485fb5e722d47690508f2a3506d30c. Nonetheless, changed in the latest force push.